src/vm/codeman.cpp

```diff
@@ -985,6 +985,13 @@ PTR_VOID GetUnwindDataBlob(TADDR moduleBase, PTR_RUNTIME_FUNCTION pRuntimeFuncti
 
     return pUnwindInfo;
 
+#elif defined(_TARGET_X86_) && defined(FEATURE_PAL)
+    PTR_UNWIND_INFO pUnwindInfo(dac_cast<PTR_UNWIND_INFO>(moduleBase + RUNTIME_FUNCTION__GetUnwindInfoAddress(pRuntimeFunction)));
+
+    *pSize = ALIGN_UP(sizeof(ULONG), sizeof(DWORD));
```

Comment:
Aligning to sizeof(BYTE) doesn't make sense, it would always return the first parameter of the macro unchanged (align ensures that the value returned by the macro is a multiple of the second parameter and sizeof(BYTE) is 1 and all numbers are multiple of 1)